mobile/src/actions/settings.js

```diff
@@ -73,6 +73,7 @@ export const registerDevice = () => async (dispatch, getState) => {
   await cozy.client.authorize().then(async ({ client }) => {
     dispatch(setClient(client))
     await cozy.client.offline.replicateFromCozy('io.cozy.files')
+    cozy.client.offline.startRepeatedReplication('io.cozy.files', 15)
```

Comment:
`replicateFromCozy have an await, so the replication is finish and we launch the repeated replication.
When I remove `replicationFromCozy` this app failed (OPEN_FOLDER_FAILURE) on first connection.